test/logs/main.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2017 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	"github.com/knative/build-pipeline/test/logs/color"
+	"github.com/knative/build-pipeline/test/logs/pipelinerun"
+	trlogs "github.com/knative/build-pipeline/test/logs/taskrun"
+
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	ns = flag.String("n", "default", "The namespace scope for this CLI request")
+	pr = flag.String("pr", "", "Name of pipelinerun to tail the logs")
+	tr = flag.String("tr", "", "Name of task run to tail the logs")
+	f  = flag.String("f", "", "Name of the file to write logs")
+)
+
+func main() {
+	flag.Parse()
+	var out io.Writer
+	// set default output as std out
+	out = os.Stdout
+
+	if (*pr == "" && *tr == "") || (*pr != "" && *tr != "") {
+		errorMsg(fmt.Sprintf("Usage: %s [-n NAMESPACE] [-pr PIPELINERUN-NAME] / [-tr TASKRUN_NAME] [-f FILE-NAME]", os.Args[0]), nil, out)
```

Comment:
nit: we may want to print a message "you can't have both `-pr`/`-tr` flags or non" maybe ?